pkg/controller/controlplane/valuesprovider.go

```diff
@@ -328,40 +327,19 @@ func (vp *valuesProvider) GetControlPlaneShootChartValues(
 }
 
 // GetControlPlaneShootCRDsChartValues returns the values for the control plane shoot CRDs chart applied by the generic actuator.
-// Currently the provider extension does not specify a control plane shoot CRDs chart. That's why we simply return empty values.
+// Currently, the provider extension does not specify a control plane shoot CRDs chart. That's why we simply return empty values.
 func (vp *valuesProvider) GetControlPlaneShootCRDsChartValues(
 	_ context.Context,
 	_ *extensionsv1alpha1.ControlPlane,
-	cluster *extensionscontroller.Cluster,
+	_ *extensionscontroller.Cluster,
 ) (map[string]interface{}, error) {
-	k8sVersionLessThan118, err := version.CompareVersions(cluster.Shoot.Spec.Kubernetes.Version, "<", gcp.CSIMigrationKubernetesVersion)
-	if err != nil {
-		return nil, err
-	}
-
 	return map[string]interface{}{
 		"volumesnapshots": map[string]interface{}{
-			"enabled": !k8sVersionLessThan118,
+			"enabled": true,
```

Comment:
Is it possible to change the default value to true here
https://github.com/gardener/gardener-extension-provider-gcp/blob/310dd84ac7159cb4009d429bf55d9af5e48ddb86/charts/internal/shoot-crds/values.yaml#L1-L2
and to drop this `GetControlPlaneShootCRDsChartValues` func?

We could also adapt https://github.com/gardener/gardener-extension-provider-gcp/blob/9c55612de3c5bf27e4f7086a4b39d826aa7b7e30/charts/internal/seed-controlplane/values.yaml#L6-L7 - the csi-driver-controller chart to be enabled by default.